src/node_mgmt/commands.py

```diff
@@ -285,23 +292,26 @@ def sys_reboot(node):
 
 def sys_start_snapd(node):
     os.system(
-        'busctl call org.freedesktop.systemd1 /org/freedesktop/systemd1 org.freedesktop.systemd1.Manager StartUnit ss "snapd.service" "replace"'
+        'busctl call org.freedesktop.systemd1 /org/freedesktop/systemd1 org.freedesktop.systemd1.Manager StartUnit ss '
+        '"snapd.service" "replace"'
     )
 
 
 def sys_stop_snapd(node):
     os.system(
-        'busctl call org.freedesktop.systemd1 /org/freedesktop/systemd1 org.freedesktop.systemd1.Manager StopUnit ss "snapd.service" "replace"'
+        'busctl call org.freedesktop.systemd1 /org/freedesktop/systemd1 org.freedesktop.systemd1.Manager StopUnit ss '
+        '"snapd.service" "replace"'
     )
 
 
 def sys_remount_rw(node):
     os.system(
-        'busctl call org.freedesktop.systemd1 /org/freedesktop/systemd1 org.freedesktop.systemd1.Manager StartUnit ss "remount-rw.service" "replace"'
+        'busctl call org.freedesktop.systemd1 /org/freedesktop/systemd1 org.freedesktop.systemd1.Manager StartUnit ss '
+        '"remount-rw.service" "replace"'
     )
 
 
 def sys_remount_ro(node):
     os.system(
-        'busctl call org.freedesktop.systemd1 /org/freedesktop/systemd1 org.freedesktop.systemd1.Manager StopUnit ss "remount-rw.service" "replace"'
+        'busctl call org.freedesktop.systemd1 /org/freedesktop/systemd1 org.freedesktop.systemd1.Manager StopUnit ss '
```

Comment:
I think something weird happened with linting here, and the tail of the command was actually truncated (rather than split to the next line)

Reply:
ah, that's definitely my bad